train_all_models.sh

```diff
@@ -47,3 +47,6 @@ scripts/train_cifar100_qat8_effnet2.sh "$@"
 echo "-----------------------------"
 echo "Training Tiny SSD model with SVHN"
 scripts/train_svhn_tinierssd.sh "$@"
+echo "-----------------------------"
+echo "Training Tiny SSD model with FaceDet"
```

Comment:
"with FaceDet" gives the wrong impression that the FaceDet is the name of the dataset. We need something like "Training Tiny SSD face detection model"..

Reply:
I agree, changed the statement to "Training Tiny SSD face detection model" with the new commit.